curriculum/challenges/english/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast.md

```diff
@@ -77,7 +77,7 @@ assert(code.match(/background-color:\s*?hsl\(120,\s*?25%,\s*?55%\)/gi));
   </header>
   <article>
     <h2>A Word on the Recent Catnip Doping Scandal</h2>
-    <p>The influence that catnip has on feline behavior is well-documented, and its use as an herbal supplement in competitive ninja circles remains controversial. Once again, the debate to ban the substance is brought to the public's attention after the high-profile win of Kittytron, a long-time proponent and user of the green stuff, at the Claw of Fury tournament.</p>
+    <p>The influence that catnip has on feline behavior is well-documented, and its use as a herbal supplement in competitive ninja circles remains controversial. Once again, the debate to ban the substance is brought to the public's attention after the high-profile win of Kittytron, a long-time proponent and user of the green stuff, at the Claw of Fury tournament.</p>
```

Comment:
You didn't revert  even if you said it?

Reply:
Yeah, I wasn't totally sure of the best way to do that. So I rather pushed another commit which changed it back, it's a bit messy i know.

Reply:
I don't think the commit targeted this tho, this is the final result of the PR

Reply:
As far as I can see when I check the repo, the file is showing "an" and when I click edit file from the 3 dots menu here, the file is also already showing "an".

Reply:
But it's not, the current final version is with an "a"
```suggestion
    <p>The influence that catnip has on feline behavior is well-documented, and its use as an herbal supplement in competitive ninja circles remains controversial. Once again, the debate to ban the substance is brought to the public's attention after the high-profile win of Kittytron, a long-time proponent and user of the green stuff, at the Claw of Fury tournament.</p>
```